src/components/form/TextInput/TextInput.tsx

```diff
@@ -248,6 +251,31 @@ export const TextInput = forwardRef<HTMLDivElement, TextInputProps>(
                 : {}),
             ...InputProps,
           }}
+          sx={
+            resetStyle
```

Comment:
What about having the `props.sx` applied if present, otherwise applying the default style written here.
That would allow removing this new props and only places that needs a custom style can define them on invocation.

Also, can we have this "reset" style in the MUI config or does it conflict with another component using the input?

Reply:
Not sure to understand your point, do you mean that all invocations should pass a props `sx` instead of `resetStyle` ?

Reply:
You should apply the sx props only if it's passed. Otherwise (in all places you use `resetStyle` in invocation), the default style will apply by default.

Reply:
Discussed together, this attribute cannot be factorised so better to keep such approach. We'll think about maybe a better naming but keep the props way